### PR TITLE
Added Saas Form 

### PR DESCRIPTION
**Pull request by sonamgupta21**
Netlfiy Link : https://siglens.netlify.app/
image
image

### CODE REVIEW

File: form-submit.html [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+          <a href="/index.html" id="install-siglens-btn"><i class="fa fa-arrow-left mr-2" aria-hidden="true"></i>Go
+            Back</a>
+        </li>
+      </div>
+    </ul>
+    <div class="d-flex justify-content-center align-items-center flex-column mt-5 pt-md-2 ">
+      <div class="thank-tick d-flex align-items-center justify-content-center ">
+        <i class="fa-solid fa-check text-white"></i>
+      </div>
+      <h1 class="main-heading text-center text-uppercase mt-5">
```
Comment (sunitakawane) [inline form-submit.html:116]: this font looks a bit big. Also do we want to say that "We will get in touch"

File: index.html [hunk 2]
```diff
@@ -467,6 +467,29 @@ <h3>SigLens vs InfluxDB/Parquet/Arrow</h3>
     </div>
   </section>
 
+  <section class="my-4 pb-5">
```
Comment (sunitakawane) [inline index.html:470]: where are we saving this information?

Comment (sonamgupta21) [reply index.html:470]: We are saving it in the Airtable database.

File: form-submit.html (hunk 1 above)
Comment (sonamgupta21) [reply form-submit.html:116]: Is this okay?
image

**Review by sunitakawane — APPROVED** [review]

**Pull request merged**